CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ AppEngine version, listed here to ease deployment and troubleshooting.
 
 ## Next Release (replace with git tag when deployed)
 
+## `20200714t110212-all`
+
```

Comment:
Nit: Maybe add a note on the new UI enabled?

Reply:
Yeah, should have done, too late. I already pushed deploy :D